Makefile

```diff
@@ -10,7 +10,8 @@ INSTALL=install
 CFLAGS ?= -O2
 PICFLAG = -fPIC
 C99FLAG = -std=c99
-UCFLAGS = $(CFLAGS) $(PICFLAG) $(C99FLAG) -DUTF8PROC_EXPORTS
+WCFLAGS = -Wall -Wmissing-prototypes -pedantic
```

Comment:
It's not included in `-Wall` [for gcc](https://gcc.gnu.org/onlinedocs/gcc-6.1.0/gcc/Warning-Options.html#Warning-Options).